src/configs/env.ts

```diff
@@ -459,7 +459,7 @@ export const ENABLE_DEFAULT_PLAN_USER_FILTER =
   process.env.REACT_APP_ENABLE_DEFAULT_PLAN_USER_FILTER === 'true';
 export type ENABLE_DEFAULT_PLAN_USER_FILTER = typeof ENABLE_DEFAULT_PLAN_USER_FILTER;
 
-export const TASK_GENERATION_STATUS = process.env.REACT_APP_TASK_GENERATION_STATUS;
+export const TASK_GENERATION_STATUS = process.env.REACT_APP_TASK_GENERATION_STATUS || 'ignore';
```

Comment:
is setting as `ignore` here a convention?

Reply:
if so, can we use the constant here

Reply:
setting ignore is not a convention.